docs/how-to-open-a-file.rst

```diff
@@ -87,6 +87,34 @@ For further examples which deal with files held on typical cloud services please
 
 
 
+----------
+
+
+Opening Django Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Django implements a `File Storage API <https://docs.djangoproject.com/en/5.1/ref/files/storage/>`_ to store files. The default is the `FileSystemStorage <https://docs.djangoproject.com/en/5.1/ref/files/storage/#the-filesystemstorage-class>`_, but the `django-storages <https://django-storages.readthedocs.io/en/latest/index.html>`_ library provides a number of other storage backends.
+
+You can open the file, move the contents into memory, then pass the contents to |PyMuPDF| as a stream.
+
+.. code-block:: python
+
+    import pymupdf
+    from django.core.files.storage import default_storage
+
+    from .models import MyModel
```

Comment:
Where does "MyModel" come from - is that part of Django?

Reply:
Yes, so `MyModel` is my made up representation of a database table.